### ISSUE TITLE
Display rules around stock sales

### ISSUE THREAD

**Issue by benjaminxscott** [tobymao/18xx#10063]
**What's your idea?**

Games have different rules around when shares may be sold, typically "not until a company has operated" if there is such a restriction. It would be nice to have that info shown somewhere, perhaps on the Market and/or Info tabs

**Are there related ideas?**

I didn't see any on a visual grep of Issues

**Comment by philcampeau** [comment]
I'm not so sure about this. Games also have different rules about tile lays (how many you can lay, permissive/semi-restrictive), tokening, full-pay/half-pay, ownership limit, bankpool limit, sold out price bumps, etc. If we were to include all of that information, things would get pretty cluttered pretty fast.

The link to the rules is in the Info tab already. At a certain point, players should be reading the rules if they want to know how the game works.

**Comment by benjaminxscott** [comment]
I see what you're saying re clutter, perhaps making it contextual?

We have `status_str(corp)` which would add a little blurb at the bottom of corp card about "only president may sell until operates", so that would only be added when relevant

https://github.com/benjaminxscott/18xx/blob/40159619f5bee3ce2bbfedeb3276a70d08e1b7d2/lib/engine/game/base.rb#L2971
